src/GitVersion.Core/Helpers/DateFormatter.cs

```diff
@@ -0,0 +1,31 @@
+using System.Globalization;
+
+namespace GitVersion.Helpers;
+
+internal class DateFormatter : IValueFormatter
+{
+    public int Priority => 2;
+
+    public bool TryFormat(object? value, string format, out string result)
+    {
+        result = string.Empty;
+
+        if (value is DateTime dt && format.StartsWith("date:"))
```

Comment:
Is the `date:` prefix required? Won't format strings apply to the object's type transparently?